### PR TITLE
[SE-4473] Add completion aggregator URL

### PR DESCRIPTION
**Pull request by pkulkark**
This PR adds the completion aggregator URL to config, which will need to be overriden inorder to display the progress bar.
This is dependent on [edx-platform#29271](https://github.com/edx/edx-platform/pull/29271).

## Testing Instructions:
As described in [edx-platform#29271](https://github.com/edx/edx-platform/pull/29271)

## Reviewers

- [ ] @gabor-boros 
- [ ] edX reviewer: TBD

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pkulkark! I've created [OSPR-6211](https://openedx.atlassian.net/browse/OSPR-6211) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by gabor-boros** [comment]
👍  🎉 This looks good to me. @natabene  this is ready for edX review I believe.

- [x] I tested what's in the test instructions
- [x] I read through the code

**Comment by natabene** [comment]
@pkulkark Thank you for your contribution.

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -1068,6 +1068,9 @@ EDXAPP_COPY_CONFIG_ENABLED: "{{ COMMON_COPY_CONFIG_ENABLED }}"
 # Dashboard courses default limit
 EDXAPP_DASHBOARD_COURSE_LIMIT: null
 
+# Completion Aggregator URL needed for progress bar
+EDXAPP_COMPLETION_AGGREGATOR_URL: null
```
Comment (jdmulloy) [inline playbooks/roles/edxapp/defaults/main.yml:1076]: I think this needs to be referenced somewhere else to take effect. Similar to how ```EDXAPP_DASHBOARD_COURSE_LIMIT``` is referenced inside ```generic_env_config```, although I don't know if that's the right place or not.

Comment (pkulkark) [reply playbooks/roles/edxapp/defaults/main.yml:1076]: @jdmulloy You're right! I missed that. Added it now. Thanks.

Comment (loucicchese) [reply playbooks/roles/edxapp/defaults/main.yml:1076]: @pkulkark This looks good, thanks for addressing @jdmulloy 's comment.  He is currently out on paternity leave, so I am filling in here.

**Review by loucicchese — APPROVED** [review]
LGTM - I believe we don't want to merge until the edx-platform PR is merged as well(?).
https://github.com/edx/edx-platform/pull/29271

**Comment by pkulkark** [comment]
@loucicchese The corresponding edx-platform PR has been merged - https://github.com/openedx/edx-platform/pull/29271. This can now be merged as well.

**Comment by natabene** [comment]
@jdmulloy @loucicchese Can you merge this now or would you prefer another support ticket for it?

**Comment by openedx-webhooks** [comment]
@pkulkark 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/configuration&entry.867055334=https://github.com/openedx/configuration/pull/6609&entry.1484655318=https://github.com/pkulkark&entry.752974735=2021-11-08+11:38&entry.1917517419=2022-03-18+19:25&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**